doc/rst/source/plot_common.rst_

```diff
@@ -70,6 +70,7 @@ Optional Arguments
 **-D**\ *dx*/*dy*
     Offset the plot symbol or line locations by the given amounts *dx/dy*
     [Default is no offset]. If *dy* is not given it is set equal to *dx*.
+    You may append dimensional units from **c**\ \|\ **i**\ \|\ **p** to each value.
```

Comment:
Please clarify also which *c* or *p* is the default unit. That is **-D4** means 4 cm or 4 points?